src/Compilers/CSharp/Test/Syntax/project.json

```diff
@@ -2,6 +2,7 @@
   "dependencies": { },
   "frameworks": {
     "netstandard1.3": {
+      // xunit isn't compat with netstandard
```

Comment:
Parts of our toolchain in the past have not supported comments here.  @agocke is right that we need to remove them else feel the pain later.